students/daniel_carrasco/session03/mailroom.py

```diff
@@ -1,35 +1,52 @@
 #!/usr/bin/env python3
 def main():
-
-    print('\noptions\n')
-    print('A = Send a Thank You')
-    print('\n')
-
-    response = input('what do you want to do?').upper()
-    if response == 'A':
+    print('\nChoose an action(1 - 3):\n\n\
```

Comment:
There's not `while True` loop here which i think is why you were calling the `main()` in the else.  You should use the while true instead.